sphinx/themes/basic/static/searchtools.js

```diff
@@ -394,7 +398,12 @@ const Search = {
       return acc;
     }, []);
 
-    results = results.reverse();
+    return results.reverse();
+  },
+
+  query: (query) => {
+    const searchTerms = Search._parseQuery(query);
```

Comment:
It's not just "searchTerms" but rather "[query, searchTerms, excludedTerms, highlightTerms, objectTerms]". Either you deconstruct the result and call explicitly the next function or you change the variable name (sorry I didn't spot it in the other PR).